### PR TITLE
[Edit] clear.md added PNG example

### PR DESCRIPTION
**Pull request by DanWSDev**
### Description
Created and added a PNG image for the code example.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by DanWSDev** [comment]
> @DanWSDev the image `css-display-clear-eaxmple.png` that you've added, doesn't need the title attribute that reads "Moves the img element below the h1 element" given to the image. Please remo


> @DanWSDev the image `css-display-clear-eaxmple.png` that you've added, doesn't need the title attribute that reads "Moves the img element below the h1 element" given to the image. Please remove it.

@ishg-153 Updated the image to remove the text

**Review by ishg-153 — APPROVED** [review]
Looks good for a second review.

File: content/css/concepts/display/terms/clear/clear.md [hunk 1]
```diff
@@ -42,3 +42,5 @@ Making an `img` element move down a _left_ floated `h1` element.
   clear: left;
 }
 ```
+
+![css-display-clear-example](https://github.com/DanWSDev/docs/blob/patch-4/media/css-display-clear-example.png)
```
Comment (SSwiniarski) [inline content/css/concepts/display/terms/clear/clear.md:46]: You need to use the URL as specified in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting).

File: content/css/concepts/display/terms/clear/clear.md [hunk 2]
```diff
@@ -42,3 +42,5 @@ Making an `img` element move down a _left_ floated `h1` element.
   clear: left;
 }
 ```
+
```
Comment (SSwiniarski) [inline content/css/concepts/display/terms/clear/clear.md:45]: Add a sentence introducing the image and explaining it is an example of the above CSS.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@DanWSDev I did a second review and had a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@DanWSDev looks good.

**Pull request merged**